concepticondata/conceptlists.tsv

```diff
@@ -87,7 +87,7 @@ Gabelentz-1891-120	Gabelentz, Hans Georg Conon	1891		121	historical	German	Globa
-Ardila-2007-40	Ardila, Alfredo	2007		40	naming test	English,Spanish,Esperanto,Latin,Turkish,Basque,Russian,Sikuani	Global	http://dx.doi.org/10.1016/j.acn.2007.01.016	Ardila2007	Ardila2007	This list was created as an alternative to current *naming tests* which are used in medicine to determine degrees of aphasia and for similar purposes. For this reason, the tests are usually accompanied by images, and the same holds for [this test](http://ars.els-cdn.com/content/image/1-s2.0-S088761770700011X-gr1.jpg), which, in contrast to earlier tests, like the Boston Naming Test, uses the idea of Swadesh lists as a new starting point for the creation of a test list. According to the author, the list has 38 items from the list of [Swadesh (1955)](:ref:Swadesh-1955-100) and 2 items from the list of [Swadesh (1952)](:ref:Swadesh-1952-200), namely &quot;snake&quot; and &quot;worm&quot;. The item selection was based on different factors, including recognizability on images, a coverage of five core semantic categories. Note that the list we give here was merged from two tables in the original paper (one containing the test items, the other translations in additional languages).	300,305	
+Ardila-2007-40	Ardila, Alfredo	2007		40	naming test	English,Spanish,Esperanto,Latin,Turkish,Basque,Russian,Sikuani	Global	http://dx.doi.org/10.1016/j.acn.2007.01.016	Ardila2007	Ardila2007	This list was created as an alternative to current *naming tests* which are used in medicine to determine degrees of aphasia and for similar purposes. For this reason, the tests are usually accompanied by images, and the same holds for [this test](http://ars.els-cdn.com/content/image/1-s2.0-S088761770700011X-gr1.jpg), which, in contrast to earlier tests, like the Boston Naming Test, uses the idea of Swadesh lists as a new starting point for the creation of a test list. According to the author, the list has 38 items from the list of [Swadesh (1955)](:ref:Swadesh-1955-100) and 2 items from the list of [Swadesh (1952)](:ref:Swadesh-1952-200), namely &quot;snake&quot; and &quot;worm&quot;. The item selection was based on different factors, including recognizability on images, a coverage of five core semantic categories. Note that the list we give here was merged from two tables in the original paper (one containing the test items, the other translations in additional languages).	300305	
```

Comment:
sorry @evoling , for this late code review, but the poznan meeting took some time. You see in this case, that 300,305 changed from 300305. The actual meaning is: the list is on page 300 and on page 305. I assume, your spreadsheet editor just turned it into some numerical thing. We can either merge and add it, or, if not too annoying for you: could you re-adjust this part?